Lab1.ipynb

```diff
@@ -1,11 +1,623 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(1, 1) # create a figure\n",
+    "ax.hist(r, 100, density=True) # fill the figure with a hist & Rayleigh distribution r\n",
+    "plt.tick_params(labelsize = 24) # sets tick parameter for the plot, in this case, the label size\n",
+    "plt.xlim([-0.5,5]) # sets rage for x axis\n",
+    "x = np.linspace(-0.5,5,1000) # create an array for x values with numpy that contains equally spaced 1000 points between -0.5 and 5\n",
+    "ax.plot(x, rayleigh.pdf(x, 0, 1), linewidth = 8,alpha = 0.7) \n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### A 3 A"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Suppose the Rayleigh distribution plotted above is my signal-free data. I choose the value when x = 2.5 for my hypothetical measurement. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### A 3 B"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The statistical question I want to ask in words is: If there was no signal, and the signal-free data has a Rayleigh distribution (null-hypothesis), what is the probability that the data produces a signal that is equally or more signal-like than what I observed?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### A 3 C"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now I want to convert the statistical question in section A 3 B into a mathematical integral. \n",
+    "To find the probability asked in A 3 B, we need to integrate the probability density function of the Rayleigh distribution from our candidate x = 2.5 to positive infinity."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### A 3 D"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The code in the next cell uses the survival function, which is defined as 1 - the cumulative distribution function(cdf), to calculate the probability that the background produced the signal. <br>\n",
+    "The cumulative distribution function takes the integral from negative infinity to our candidate. As I mentioned in the previous section, we want to integrate the probability density function of the Rayleigh distribution from our candidate x = 2.5 to positive infinity. Therefore, the survival function, 1 - the cumulative distribution function, will give us the correct value of probability."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The probability that the background produced the signal at x = 2.5 is 0.04393693362340742.\n"
```

Comment:
Since Rayleigh distribution is a continuous distribution, i think the probability you calculate is not a probability at one specific point but it is a probability within a set region.